src/components/Button.js

```diff
@@ -8,16 +8,23 @@ class Button extends Component {
     }
 
     render(){
+        const buttonStyle = {
+            backgroundColor: this.props.color,
+            width: this.props.wide ? "50%":"25%"
+        };
+
         return(
-            <button className="Button">
+            <button className="Button" style={buttonStyle}>
                 {this.props.name}
             </button>
         )
     }
 }
 
 Button.propTypes = {
-    name: PropTypes.string.isRequired
+    name: PropTypes.string.isRequired,
+    color: PropTypes.string.isRequired,
+    wide: PropTypes.bool.isRequired
 }
 
```

Comment:
Hi, using default props to set the colour for  orange buttons is a requirement.
* [ ] defualt color for button is orange.

for the buttons that are orange, don't send a color prop but let the default color kick in.

Reply:
Added the defaul prop